### PR TITLE
New app: rtorrent-rutorrent-cross-seed

### PR DESCRIPTION
**Pull request by ac1dburnz**
**Description**

⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ✔️] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
Testing was done in https://github.com/truecharts/charts/pull/11594  but this needs to be pulled 
-->

**📃 Notes:**

**✔️ Checklist:**

- [✔️ ] ⚖️ My code follows the style guidelines of this project
- [✔️ ] 👀 I have performed a self-review of my own code
- [✔️ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [✔️ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ✔️] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: mirror/rtorrent-rutorrent-cross-seed/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM docker pull ghcr.io/ac1dburnz/rtorrent-rutorrent-cross-seed:latest@sha256:018a0edda145c62fc6400a7de9d901a53c384d7fe6a4e928e0163c97c4bc8f1f
```
Comment (xstar97) [inline mirror/rtorrent-rutorrent-cross-seed/Dockerfile:1]:
```suggestion
FROM ghcr.io/ac1dburnz/rtorrent-rutorrent-cross-seed:latest@sha256:018a0edda145c62fc6400a7de9d901a53c384d7fe6a4e928e0163c97c4bc8f1f
```

**Review by xstar97 — CHANGES_REQUESTED** [review]

**Comment by stavros-k** [comment]
Crappy image source, they don’t have versioned tag. 
pff, another thing that will contribute to rate limits -.-

**Comment by ac1dburnz** [comment]
@stavros-k  i can redo it with a version tag if youd prefer it

**Comment by ac1dburnz** [comment]
or i can do it for the next update

**Pull request merged**